### PR TITLE
Remove signing

### PR DESCRIPTION
**Pull request by generik0**
#22

### CODE REVIEW

File: src/Serilog.Settings.AppSettings/Serilog.Settings.AppSettings.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>XML configuration (System.Configuration &lt;appSettings&gt;) support for Serilog.</Description>
-    <VersionPrefix>2.2.0</VersionPrefix>
+    <VersionPrefix>2.2.2</VersionPrefix>
```
Comment (nblumhardt) [inline src/Serilog.Settings.AppSettings/Serilog.Settings.AppSettings.csproj]: The version doesn't need to be update for -dev builds; I'll tweak this and merge now.

**Comment by nblumhardt** [comment]
Thanks!

**Comment by nblumhardt** [comment]
Argh, oops - accidentally accepted the default commit message, which was "Remove Signing" - this isn't in fact what the commit does - it should read "Remove Delay Signing" - i.e., make sure the package is signed during build.

**Comment by generik0** [comment]
Your right. The reason for the bump is that i have placed the nuget (without dev) on my own myget feed.
I was getting warnings that i shouldn't be making nuget packages from pre release versions.
To much yellow. So the bump was done for my future nugetting/resoving.
Was i thinking to much about myself? Well, 2.2.2 is also a cool version.

**Pull request merged**